src/Notebook/notebook-help.tsx

```diff
@@ -74,6 +74,22 @@ function ColumnSelectionHelp() {
   );
 }
 
+function AnonParamsSelectionHelp() {
+  return (
+    <div>
+      <Title level={4}>Anonymization Parameters</Title>
+      <Paragraph>
+        The default set of parameters ensures correct anonymization and has maximum analytical utility. It is
+        recommended for the majority of use cases. Select stronger anonymization, if it does not lower the analytical
```

Comment:
"Select stronger anonymization, if it does not lower the analytical..." I feel like the comma makes this like a suggestion instead of a condition.

Reply:
thanks, without comma is actually the only correct punctuation in English, I always get it wrong :shrug: